tests/testsuite/read_lockfile.rs

```diff
@@ -112,40 +102,31 @@ fn package_with_transitive_deps() {
             str![[r#"
 [
   {
-    "metadata": null,
-    "package": [
-      {
-        "checksum": "3a351dafbc8a3a9cba7c06dfe8caa11a3a45f800a336bb5b913a8f1e2652d454",
-        "dependencies": null,
-        "name": "a",
-        "replace": null,
-        "source": "registry+https://github.com/rust-lang/crates.io-index",
-        "version": "1.0.0"
-      },
-      {
-        "checksum": "ee3b274199c39817bfb6018e6cbe07ca43dd18241c42400d800e2545b77fb23b",
-        "dependencies": [
-          "a"
-        ],
-        "name": "b",
-        "replace": null,
-        "source": "registry+https://github.com/rust-lang/crates.io-index",
-        "version": "1.0.0"
-      },
-      {
-        "checksum": null,
-        "dependencies": [
-          "a",
-          "b"
-        ],
-        "name": "read-lockfile-test",
-        "replace": null,
-        "source": null,
-        "version": "0.1.0"
-      }
-    ],
-    "root": null,
-    "version": 4
+    "reason": "lockfile",
+    "lockfile": {
+      "package": [
```

Comment:
Should we do a message per locked package?

Been thinking over this with libtest's json output and I feel like it might be easier to extend a schema by emitting more messages rather than cramming more information into existing messages